rules/show-change-in-reports/rule.md

```diff
@@ -0,0 +1,35 @@
+---
+type: rule
+archivedreason:
+title: Data Layout - Do you show change in your reports?
+guid: 7a272cbc-bdb5-44f7-9877-7e8d38690d1c
+uri: show-change-in-reports
+created: 2023-12-11T14:38:33.0000000Z
+authors: 
+  - title: Jeoffrey Fischer
+    url: https://ssw.com.au/people/jeoffrey-fischer
+related:
+- customization-do-you-know-which-version-of-sql-reporting-services-and-visual-studio-you-are-using
+redirects: []
+
+---
+
```

Comment:
maybe we should add an intro?

Reply:
Yes - actually all SSRS rules don't have an intro yet
I will create a PR now to include an intro to the SSRS rules